### DISCUSSION TITLE
Is it possible to search a pattern inside the preview window of bat?

### DISCUSSION THREAD

**Discussion by carpetaa** [junegunn/fzf#2806] [Q&A] [answered]
I can search some pattern inside the bat window with `/` when using bat like this.
`bat <some file>` 

Then how can I use this searching function of bat in the preview window of fzf when using them like below?
`fzf --preview 'bat --style=numbers --color=always --line-range :500 {}'`

**Comment by junegunn** [comment, marked as answer]
No, preview window is not interactive.

You can use `execute` action to open an interactive bat process.

```sh
fzf --preview 'bat --style=numbers --color=always {}' --bind 'ctrl-o:execute(clear; bat --paging=always {})'
```

**Reply by carpetaa** [reply]
This command works very well.
Thanks.
